install-3scale-rwo/system-storage-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  # This name uniquely identifies the PVC. Will be used in deployment below.
+  name: system-storage
+  namespace: "3scale-rwo"
+  labels:
+    app: 3scale-api-management
+    threescale_component: system
+spec:
+  accessModes:
+    - ReadWriteOnce
```

Comment:
The whole point of creating an RWO PVC is so that you do not have to use NooBaa. Either leave this PVC out of the instructions or remove the NooBaa instructions.

Reply:
This does not belong here. Removed it